mobileChatExamples/iOSChatExample/iOSChatExample/Common/Config.swift

```diff
@@ -11,11 +11,36 @@ import AWSConnect
 
 class Config {
     
-    let startChatEndPoint: String = ""
-    let instanctId: String = ""
-    let contactFlowId: String = ""
-    let region: AWSRegionType =  // .USEast1/.USWest1 :https://docs.aws.amazon.com/general/latest/gr/rande.html
-    let agentName = ""
-    let customerName = ""
+//    let startChatEndPoint: String = ""
```

Comment:
Do we want to commit this? If this is intentional and supposed to be like a template, I would add more comments clarifying what this is and when/how to use it.